low-level/framework/messaging/egress_accumulated_msgs_processor.py

```diff
@@ -74,9 +74,15 @@ def initialize(self, conf_reader, msgQlist, products):
         self.store_queue = StoreQueue()
         self._read_config()
         producer_initialized.wait()
-        self._producer = MessageProducer(producer_id="acuumulated processor",
-                                         message_type=self._message_type,
-                                         method=self._method)
+        self.create_MsgProducer_obj()
+
+    def create_MsgProducer_obj(self):
+        self._producer = None
+        try:
+            self._producer = MessageProducer(producer_id=self._producer_id,
```

Comment:
self._producer_id
where this is declared

Reply:
Inside _read_config method.
line 157.

Reply:
got it